### DISCUSSION TITLE
Multiple Plot Spectrum Pages

### DISCUSSION THREAD

**Discussion by hovavalon** [audacity/audacity#2946] [Ideas & Feature Requests]
Currently, it is impossible to present the spectra of two different sections at the same time using the "Plot Spectrum" option.
If it would be possible it would be very helpful.

**Comment by LWinterberg** [comment]
You can click the Export button in the plot spectrum window to get a TSV file of the values. From there, you can use Excel or similar (or a python or R program) to draw the graphs you want

**Reply by hovavalon** [reply]
Yes, but this is time consuming and not comfortable.
I agree it's not an important feature, but for some cases it would improve audacity's usability.

**Reply by asher-bs** [reply]
The two files maybe not sync so  I think the better solution is to to add the option to "hold" the spectrum of a files (like in Audition)
